src/components/HacksPage.js

```diff
@@ -65,7 +65,7 @@ const Hacks = ({ dispatch }) => {
     dispatch(fetchUsers());
   }, [dispatch]);
   const renderHacks = () => {
-    return data.map(hack => {
+    data.map(hack => {
```

Comment:
You'll need that `return` if you want to see anything on that page... or remove the `{` and `}` from the outer block (eg `const renderHacks = () => data.map(hack => {`